src/stylesheets/components/Auth/_logo-auth.scss

```diff
@@ -0,0 +1,25 @@
+.logo-auth {
+  $self: &;
+
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 24px;
+
+  &__img {
+    align-self: center;
+    height: 40px;
+    margin: 0 24px 24px;
+    width: 167.84px;
```

Comment:
Do need to hard-coded the image size?

Reply:
I fixed the size of this logo as it is in the figma design 😅

Reply:
Could we leave it as default, I think it will take the original width of the image.